python/desc/imsim/imSim.py

```diff
@@ -8,7 +8,16 @@
 from collections import namedtuple, defaultdict
 import logging
 import gc
-import ConfigParser
+
+# python_future no longer handles configparser as of 0.16.
+# This is needed for PY2/3 compatabiloty.
+try:
```

Comment:
I'd prefer to do something like this in order to retain namespace scoping below:
```
try:
    import configparser
except ImportError:
    # python 2 backwards-compatibility
    import ConfigParser as configparser
```